src/ui/ClinicalPanel/index.tsx

```diff
@@ -263,6 +265,7 @@ type ClinicalPanelProps = {
     hasClinicalInfo: boolean;
     setInteractiveMode: (interactiveMode: boolean) => void;
     setIsClinicalPanelOpen: (isClinicalPanelOpen: boolean) => void;
+    setSelectedSvId: (selectedSv: any) => void;
```

Comment:
It would be good to avoid using `any` to fully utilize TS. Is it either `string` or `undefined`?

```suggestion
    setSelectedSvId: (selectedSv?: string) => void;
```